lib/model/rofolder.go

```diff
@@ -20,7 +20,7 @@ func init() {
 
 type sendOnlyFolder struct {
 	folder
-	cfg config.FolderConfiguration
+	config.FolderConfiguration
 }
```

Comment:
I have a funny joke about where exactly you can embed your config...

:trollface: anyway, that folder thing is a bit of a weird appending as it has almost no functionality. Should sort that out same day.